internal/daemon/controller/handlers/targets/target_service.go

```diff
@@ -447,6 +447,12 @@ func (s Service) UpdateTarget(ctx context.Context, req *pbs.UpdateTargetRequest)
 		}
 	}
 
+	// since we resolved the alias to an id, transform the request to get the
+	// correct subtype attributes
+	if err := subtypes.TransformRequest(ctx, req); err != nil {
+		return nil, errors.Wrap(ctx, err, op)
+	}
```

Comment:
This is unconditional right now, would it not normally go inside the `else` above?